feature/sessions/src/main/java/io/github/droidkaigi/confsched2023/sessions/section/TimetableList.kt

```diff
@@ -94,9 +96,13 @@ fun TimetableList(
                             text = timetableItem.startsTimeString,
                             fontWeight = FontWeight.Medium,
                         )
-                        Text(text = "|")
+                        Text(text = "|",
+                            color = md_theme_light_outlineVariant,
+                        )
```

Comment:
Could you put this back?  This is because there is a different PR that will cause conflict 🙏